mnelab/model.py

```diff
@@ -443,6 +443,31 @@ def epoch_data(self, events, tmin, tmax, baseline):
         self.current["events"] = self.current["data"].events
         self.current["name"] += " (epoched)"
 
+    @data_changed
+    def convert_od(self):
+        self.current["data"] = mne.preprocessing.nirs.optical_density(
+            self.current["data"])
+        self.current["name"] += f" (OD)"
+        self.history.append(
+            f'data = mne.preprocessing.nirs.optical_density(data)')
+
+    @data_changed
+    def apply_tddr(self):
+        self.current["data"] = mne.preprocessing.nirs.\
+            temporal_derivative_distribution_repair(self.current["data"])
+        self.current["name"] += f" (TDDR)"
+        self.history.append(
+            f'data = mne.preprocessing.nirs.'
+            f'temporal_derivative_distribution_repair(data)')
+
+    @data_changed
+    def convert_beer_lambert(self):
+        self.current["data"] = mne.preprocessing.nirs.beer_lambert_law(
```

Comment:
Same comment here, do you think it would make sense to rename this function to `beer_lambert` directly in MNE?